### PR TITLE
Flask: add priority column to analysis table

### PR DESCRIPTION
**Pull request by Madelinehazel**
Part of #505 
Making a PR in case we want to include this in the initial migration before rollout

### CODE REVIEW

File: flask/app/models.py [hunk 1]
```diff
@@ -341,6 +341,11 @@ class AnalysisState(str, Enum):
     Cancelled = "Cancelled"
 
 
+class PriorityType(str, Enum):
+    ClinicalPriority = "ClinicalPriority"
+    ResearchPriority = "ResearchPriority"
```
Comment (kevinlul) [inline flask/app/models.py:346]: Including `Priority` in the value of the enum seems kind of redundant when both the column and the enum type are labelled as such.

**Review by kevinlul — CHANGES_REQUESTED** [review]

File: flask/app/models.py (hunk 1 above)
Comment (kevinlul) [reply flask/app/models.py:346]: This also changes the number of enumerations available in `/api/enums`

Comment (Madelinehazel) [reply flask/app/models.py]: fixed in e635324d3c3643d525731b5628c95a5dcd529723

**Review by kevinlul — APPROVED** [review]
LGTM!

**Pull request merged**